src/plugins/intel_cpu/src/graph.cpp

```diff
@@ -1622,26 +1683,52 @@ void Graph::EnforceBF16() {
             continue;
 
         if (node->getType() != Type::Input && node->getType() != Type::Output) {
+            // FP16 is only implemented on limited types of node.
```

Comment:
BF16 is implemented for limited numeber of  nodes as well. The expection is the node itself will fallback on FP32 precision in case lower precision is not suppotred. 
Could you please clarify why we need different behavior for FP16?

Reply:
OK, I didn't realize that, maybe this can be removed, let me do some tests!